docker/lr-dv/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM google/deepvariant:1.3.0
+
+COPY vm_local_monitoring_script.sh /opt/
```

Comment:
Yeah, nice workaround, but wish we didn't need it.

Reply:
The Terra-as-a-service idea, after discussing with Berry, seems like a no-go.
I'm sure there are wish-list documents living somewhere, but I'd like to know what their concerns are for exposing some of these Cromwell parameters.